package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matterfront",
+  "version": "0.1.0",
+  "description": "Mattermost frontend app powered by electron",
+  "main": "./src/main.js",
+  "devDependencies": {
+    "electron-packager": "^5.1.1",
+    "electron-prebuilt": "^0.34.2"
+  },
+  "electronVersion":"0.34.1",
+  "scripts": {
+    "start":"electron ./",
+    "build": "electron-packager ./src matterfront --out=dist --ignore='^/dist$' --prune --asar --platform=all --arch=all --version=$npm_package_electronVersion"
```

Comment:
I'm planning on writing a more comprehensive build system tomorrow using `electron-packager` 's [programmatic api](https://www.npmjs.com/package/electron-packager#programmatic-api), but for now wanted to get the capability into the build for those waiting for some sort of cross-platform support.

There's enough going into that work that I wanted to make it it's own feature/PR. Felt like this would grow out-of-scope otherwise.

the issues mentioned (case issues, resources/flags) will be fixable from there.